ChessGame/src/kurs2_task2/Game/Board.java

```diff
@@ -0,0 +1,112 @@
+package kurs2_task2.Game;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.TreeMap;
+
+public class Board {
+	private TreeMap<String, Cell> board = new TreeMap<>();
+	private String[] ABC = { "A", "B", "C", "D", "E", "F", "G", "H"};
+
+
+	public TreeMap<String, Cell> getBoard() {
+		return board;
+	}
+
+	public void setBoard(TreeMap<String, Cell> board) {
+		this.board = board;
+	}
+	public Cell getCell(String cord){
+		return board.get(cord);
+	}
+	public void buildBoard() {
+		String[] tempABC = { "-A", "A", "B", "C", "D", "E", "F", "G", "H", "-H"};
+		creationCell(tempABC);
+		linkingCell(tempABC);
+		clearCell(tempABC);
+	}
+
+	public void creationCell(String[] tempABC){

```

Comment:
There is a convention that tells us to name methods with a verb. Moreover, I don't understand name of the method at all. Adjust accordingly.
hint: buildCell, createCell or something